packages/docs/docs/theme/default-theme-config.md

```diff
@@ -155,6 +155,17 @@ module.exports = {
 }
 ```
 
+### Extract Headers
+ While preparing the page, headers are extracted from the Markdown file and stored in `this.$page.headers`. By default, VuePress will extract `h2` and `h3` elements for you.
+ You can override the headers it pulls out in your `markdown` options.
+ ``` js
+module.exports = {
+  markdown: {
+    extractHeaders: [ 'h2', 'h3', 'h4' ]
+  }
+}
+```
+
```

Comment:
This option should not be placed in default theme scope @kefranabg

Reply:
you're right, I just copied this part from the [initial PR](https://github.com/vuejs/vuepress/pull/1904) 😅